google-cloud-storage/src/main/java/com/google/cloud/storage/StorageOptions.java

```diff
@@ -128,6 +129,13 @@ public static StorageOptions getDefaultInstance() {
     return newBuilder().build();
   }
 
+  /**
+   * Returns a unauthenticated {@code StorageOptions} instance.
+   */
+  public static StorageOptions getUnauthenticated() {
```

Comment:
`getUnauthenticatedInstance`?
In vacuum, I'm OK with the name, but a "parallel" method is called `getDefaultInstance` so I think it's better to have parallel names as well.